google/cloud/aiplatform/docker_utils/run.py

```diff
@@ -162,6 +187,17 @@ def run_prediction_container(
     )
     envs[_ADC_ENVIRONMENT_VARIABLE] = credential_mount_path
 
+    entrypoint = (
+        [_replace_env_var_reference(i, envs) for i in serving_container_command]
```

Comment:
Should `serving_container_command` and `serving_container_args` really be of the type Sequence or can they be Lists? If they can be List, can we default them to [] to avoid this if not None/else [] logic?

Otherwise maybe it's clearer as:
`[_replace_env_var_reference(i, envs) for i in serving_container_command or []]`

or 
`serving_container_command = serving_container_command or []`
`[_replace_env_var_reference(i, envs) for i in serving_container_command]`

Reply:
I choose the second option.

`serving_container_command` and `serving_container_args` are the type Sequence across the SDK. I would like to keep it as is.